### PR TITLE
#minor add create todolist feature

### PR DESCRIPTION
**Pull request by windy-worm**
Added the handler to create a todolist
Added shared infrastructure classes like DB contexts and endpoints

### CODE REVIEW

File: EngineBay.DemoModule/DemoModuleEndpoints.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            })
+            .WithTags(TodoListTags);
+
+            endpoints.MapGet(basePath + "/{id}", (Guid id, CancellationToken cancellation) =>
+              {
+                  return Results.Ok("todo list");
+              })
+              .WithTags(TodoListTags);
+
+            // endpoints.MapGet(
```
Comment (sahilNaran) [inline EngineBay.DemoModule/DemoModuleEndpoints.cs:27]: why not remove?

File: EngineBay.DemoModule/EngineBay.DemoModule.csproj [hunk 2]
```diff
@@ -24,23 +24,27 @@
         <AnalysisMode>AllEnabledByDefault</AnalysisMode>
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
         <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-        <IsTrimmable>true</IsTrimmable>
+        <IsTrimmable>false</IsTrimmable>
```
Comment (sahilNaran) [inline EngineBay.DemoModule/EngineBay.DemoModule.csproj:27]: Why isnt the project safe to be trimmed?

**Review by sahilNaran — APPROVED** [review]

File: EngineBay.DemoModule/EngineBay.DemoModule.csproj (hunk 2 above)
Comment (windy-worm) [reply EngineBay.DemoModule/EngineBay.DemoModule.csproj:27]: Minimal API cannot be trimmed

File: EngineBay.DemoModule/DemoModuleEndpoints.cs (hunk 1 above)
Comment (windy-worm) [reply EngineBay.DemoModule/DemoModuleEndpoints.cs:27]: will remove with the next PR

**Pull request merged**